### PR TITLE
Add trasnlation script and guide

### PR DESCRIPTION
**Pull request by bropines**
Added a script for creating translation files in the new language folder and a guide for translating strings inside files.

### CODE REVIEW

**Comment by bropines** [comment]
I checked the script and it works stably. But I came up with the guide at the last moment.

File: translation.ps1 [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            $updatedFolders += $targetDir
+        }  
+    }
+    return $updatedFolders
+}
+
+# Main script
+$languageCode = Read-Host "Please enter your language code (e.g., 'ru-ru', 'zh-cn')"
+
+$basePaths = @('G:\My Projects\Pixeval\src\Pixeval\Strings', 'G:\My Projects\Pixeval\src\Pixeval.Controls\Strings')
```
Comment (Poker-sang) [inline translation.ps1:49]: Should we use relative path here

File: translation.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   For testing, currently you need to build the program yourself. Instructions on how to build the program are described in the [Prerequisites](README.en.md#prerequisites) section of the README.en.md file.
+   Note that the libraries required for building will take up about 20-25 gigabytes. Generally, you can translate and commit your changes, and GitHub Actions will build a test version from your commit.
+
+## Translation Tools
+
+Here are some recommended tools for translating the `.resjson` files:
+
+- [Visual Studio Code](https://code.visualstudio.com/) - A powerful and customizable code editor.
+- [Sublime Text](https://www.sublimetext.com/) - A sophisticated text editor for code, markup, and prose.
+- [Notepad++](https://notepad-plus-plus.org/) - A free source code editor and Notepad replacement.
```
Comment (Poker-sang) [inline translation.md:37]: I don't think it's necessary to specify the editor, just the tool we wrote ourselves and the most common tool (VSCode) is ok

**Review by Poker-sang — CHANGES_REQUESTED** [review]

File: translation.ps1 (hunk 1 above)
Comment (bropines) [reply translation.ps1:49]: My crooked tests broke everything, now I'll fix it

**Comment by bropines** [comment]
Once again, sorry, I just tested this in two folders, and did not transfer the normal paths to the git folder

**Review by Poker-sang — APPROVED** [review]

**Pull request merged**